### DISCUSSION TITLE
Pole figure with BC-color

### DISCUSSION THREAD

**Discussion by JSLecomte** [mtex-toolbox/mtex#2196] [Ask Anything] [answered]
my question is very simple: I have an EBSD map in Band Contrast
I would like to have the associated pole figure and each orientation with the band contrast color (like example)
![question_forum_mtex_1](https://github.com/user-attachments/assets/eafe0c75-5298-47bc-89c3-47b49f32fcd5)
![question_forum_mtex_2](https://github.com/user-attachments/assets/0f65b594-2103-410d-a6f0-29975be69601)
**how to do it?**

**Comment by nyyssont** [comment, marked as answer]
Here is one way:

```matlab
mtexdata martensite
bc = ebsd('Iron bcc').bc;
rgbImage = ind2rgb(bc, colormap('gray'));
plotPDF(ebsd('Iron bcc').orientations,rgbImage,Miller(0,0,1,ebsd('Iron bcc').CS),'all');
```

![image](https://github.com/user-attachments/assets/78077d6c-ac81-4246-a891-3f1e897f6b48)

**Reply by JSLecomte** [reply]
Merci Tuomo
